### PR TITLE
add accessibility issue template

### PR DESCRIPTION
**Pull request by minrk**
for submissions of phase two in #38

### CODE REVIEW

**Comment by minrk** [comment]
You can test it out at: https://github.com/minrk/outreachy/issues/new/choose

File: .github/ISSUE_TEMPLATE/accessibility.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  - outreachy-dec22
+  - "project: improve-accessibility"
+  - "needs:review"
+body:
+  - type: markdown
+    attributes:
+      value: >
+        Thank you for improving accessibility of a Jupyter page!
+
+        If you're here, you have produced a WAVE accessibility report (as described in #39).
```
Comment (GeorgianaElena) [inline .github/ISSUE_TEMPLATE/accessibility.yml:17]:
```suggestion
        If you're here, you have produced a WAVE accessibility report (as described in https://github.com/jupyterhub/outreachy/issues/38).
```

**Review by GeorgianaElena — APPROVED** [review]
I like it

File: .github/ISSUE_TEMPLATE/accessibility.yml (hunk 1 above)
Comment (minrk) [reply .github/ISSUE_TEMPLATE/accessibility.yml]: oops, no, thinking of soemthing else. This is better!

**Review by sgibson91 — APPROVED** [review]

**Pull request merged**